lib/internal/cluster/primary.js

```diff
@@ -42,21 +42,23 @@ cluster.SCHED_RR = SCHED_RR;      // Primary distributes connections.
 let ids = 0;
 let debugPortOffset = 1;
 let initialized = false;
-
-// XXX(bnoordhuis) Fold cluster.schedulingPolicy into cluster.settings?
-let schedulingPolicy = process.env.NODE_CLUSTER_SCHED_POLICY;
-if (schedulingPolicy === 'rr')
-  schedulingPolicy = SCHED_RR;
-else if (schedulingPolicy === 'none')
-  schedulingPolicy = SCHED_NONE;
-else if (process.platform === 'win32') {
-  // Round-robin doesn't perform well on
-  // Windows due to the way IOCP is wired up.
-  schedulingPolicy = SCHED_NONE;
-} else
-  schedulingPolicy = SCHED_RR;
-
-cluster.schedulingPolicy = schedulingPolicy;
+let schedulingPolicy = SCHED_RR;
+
+const SCHEDULE_POLICY_NAME_MAP = new SafeMap([
+  ['rr', SCHED_RR],
+  ['none', SCHED_NONE]
+]);
```

Comment:
Using an array in the `Map` constructor relies on the globally mutable `Array.prototype[Symbol.iterator]` method. Consider using `.set` instead.
```suggestion
const SCHEDULE_POLICY_NAME_MAP = new SafeMap()
  .set('rr', SCHED_RR)
  .set('none', SCHED_NONE);
```